src/components/home/slider/SliderContent.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {FC} from 'react'
+import {useSlider} from 'src/context/slider'
+
+import {SliderModal} from './SliderModal'
+import * as SliderParent from './style'
+
+interface SliderContentChildProps {
+  currentSlide?: number
+  translate?: number
+}
+
+export const SliderContentChild: FC<SliderContentChildProps> = ({
+  translate = 250,
+  currentSlide = 0,
+}) => {
+  const {movies}: {movies: Movie[]} = useSlider()
```

Comment:
I HIGHLY suggest typing the return type of `useSlider` instead of declaring the return type every time you call it.

Reply:
I really wonder if there a name schema for the return value I can use?